auto-updater.js

```diff
@@ -1,6 +1,5 @@
 const { autoUpdater } = require('electron');
 const { version } = require('./package');
-const notify = require('./notify');
```

Comment:
I'd love to continue to investigate whether we really need to have this lingering require or not.